hub/powertoys/peek.md

```diff
@@ -1,15 +1,14 @@
 ---
 title: PowerToys Peek utility for Windows
 description: A system-wide utility for Windows to quickly preview file content.
-ms.date: 11/19/2024
-ms.topic: concept-article
+ms.date: 08/03/2023
+ms.topic: article
 no-loc: [PowerToys, Windows, Peek, Win]
-# Customer intent: Learn how to configure and use the Peek utility in PowerToys.
```

Comment:
Please keep the "Customer intent". We're adding this for all content that uses one of the supported content pattern types.